Library/ViewModels/SettingsViewModel.swift

```diff
@@ -675,22 +690,23 @@ private enum Notification {
     case .creatorTips:                              return "Creator tips"
     case .follower, .mobileFollower:                return "New followers"
     case .friendActivity, .mobileFriendActivity:    return "Friend backs a project"
-    case .messages, .mobileMessages:                 return "New messages"
+    case .messages, .mobileMessages:                return "New messages"
```

Comment:
Assuming these don't need translations?

Reply:
I'm assuming they don't, since these are used for tracking.